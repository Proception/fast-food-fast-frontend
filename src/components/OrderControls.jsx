import React, { Component } from 'react';

export default class OrderControls extends Component {
  render() {
    return (
      <section class="order">
        <div class="container">
          <div class="controls">
            <div class="search-box">
              <input type="text" name="" placeholder="Search Orders" />
            </div>
            <div class="search-button">
              <input class="button" type="button" name="" value="Search" />
            </div>
            <div id="sort" class="sort-controls">
              <label>Sort By : </label>
              <select>
                <option value="New" label="New">New</option>
                <option value="Processing" label="Processing">Processing</option>
                <option value="Cancelled" label="Cancelled">Cancelled</option>
                <option value="Completed" label="Completed">Completed</option>
              </select>
            </div>
          </div>
        </div>
      </section>
    )
  }
}
