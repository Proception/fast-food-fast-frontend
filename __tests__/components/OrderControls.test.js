import React from 'react';
import { shallow } from 'enzyme';
import OrderControls from '../../src/components/OrderControls.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<OrderControls />);
});
describe('OrderControls Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
