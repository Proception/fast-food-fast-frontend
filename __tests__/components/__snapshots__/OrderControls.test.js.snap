// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderControls Component should match snapshot 1`] = `
<section
  class="order"
>
  <div
    class="container"
  >
    <div
      class="controls"
    >
      <div
        class="search-box"
      >
        <input
          name=""
          placeholder="Search Orders"
          type="text"
        />
      </div>
      <div
        class="search-button"
      >
        <input
          class="button"
          name=""
          type="button"
          value="Search"
        />
      </div>
      <div
        class="sort-controls"
        id="sort"
      >
        <label>
          Sort By : 
        </label>
        <select>
          <option
            label="New"
            value="New"
          >
            New
          </option>
          <option
            label="Processing"
            value="Processing"
          >
            Processing
          </option>
          <option
            label="Cancelled"
            value="Cancelled"
          >
            Cancelled
          </option>
          <option
            label="Completed"
            value="Completed"
          >
            Completed
          </option>
        </select>
      </div>
    </div>
  </div>
</section>
`;
